app/src/common/shared/com/igalia/wolvic/ui/widgets/TrayWidget.java

```diff
@@ -818,10 +818,17 @@ private boolean updateWifiIcon(final int level) {
     }
 
     private void updateWifi() {
+        // TODO: Deprecated: Starting with Build.VERSION_CODES#S, WifiInfo retrieval is moved to ConnectivityManager
+        //  API surface. WifiInfo is attached in NetworkCapabilities#getTransportInfo() which is available via callback
+        //  in NetworkCallback#onCapabilitiesChanged(Network, NetworkCapabilities) or on-demand from
+        //  ConnectivityManager#getNetworkCapabilities(Network).
```

Comment:
Mind opening a new issue for wifi handling and link it here?

Reply:
https://github.com/Igalia/wolvic/issues/802